cloudformation/template.yaml

```
@@ -65,6 +65,9 @@ Parameters:
     Type: String
     Description: The email address where user submitted feedback notifications get sent.
     Default: ''
+    MaxLength: 254
+    AllowedPattern: ^$|^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$
+    ConstraintDescription: Malformed input - Parameter DevPortalAdminEmail must be blank or follow the format for valid email addresses, see https://www.w3.org/TR/html/sec-forms.html#email-state-typeemail
```

Let's simplify this by quite a bit. We should also be able to safely assume people know what a valid email address is - even most non-tech-literate people generally know the syntax of email addresses.

```suggestion
    ConstraintDescription: Malformed input - Parameter DevPortalAdminEmail must be blank or be a valid email address.
```